docs/api/rasa-sdk.rst

```diff
@@ -0,0 +1,90 @@
+:desc: Information about `rasa-sdk` objects and functions.
+
+.. _rasa-sdk:
+
+Rasa SDK
+========
+
+.. edit-link::
+
+Rasa SDK provides the tools you need to write :ref:`custom-actions`.
+
+.. contents::
+   :local:
+
+``Action``
+----------
+
+The ``Action`` class is the base class for any custom action. It has two methods
+that both need to be overwritten, `.name()` and `.run()`.
+
+.. _custom_action_example_verbose:
+
+In a restaurant bot, if the user says "show me a Mexican restaurant",
+your bot could execute the action ``ActionCheckRestaurants``,
+which might look like this:
+
+.. testcode::
+
+   from rasa_sdk import Action
+   from rasa_sdk.events import SlotSet
+
+   class ActionCheckRestaurants(Action):
+      def name(self) -> Text:
+         return "action_check_restaurants"
+
+      def run(self,
+              dispatcher: CollectingDispatcher,
+              tracker: Tracker,
+              domain: Dict[Text, Any]) -> List[Dict[Text, Any]]:
+
+         cuisine = tracker.get_slot('cuisine')
+         q = "select * from restaurants where cuisine='{0}' limit 1".format(cuisine)
+         result = db.query(q)
```

Comment:
Is this just pseudocode or was it taken somehwere? is there an import for `db`?

Reply:
This was copied from the `Actions` page

Reply:
eh, alright haha